### DISCUSSION TITLE
Request chart of min/avg/max input/output tokens

### DISCUSSION THREAD

**Discussion by secsilm** [langfuse/langfuse#1793] [Ideas]
### Describe the feature or potential improvement

Now we can only see the changes in the total token count, but in some scenarios, we would like to know the situation of input/output tokens.

**Comment by marcklingen** [comment]
Makes sense, this is top-of-mind already. Thanks for sharing!

**Reply by XyAgent** [reply]
This one would be very helpful!

The ability to create custom charts for evaluation run comparisons 
(Like we do when creating widgets for Dashboards)
